scripts/pi-hole/php/teleporter.php

```diff
@@ -205,6 +210,10 @@ function archive_insert_into_table($file, $table, $flush=false, $wildcardstyle=f
 		else
```

Comment:
This code is very similar to the other add/sub functions. Can this code use those functions instead?

Reply:
These routines restore from a file instead of receiving an array. We can write a wrapper that calls `add_to_table()` for the legacy import (where we only import the domains).

Reply:
It does load the domains into an array from the file, so at some point the code does become similar enough. The file loading and table flushing code can be kept in this file, but the insertion can be done through the separate method.